kmip/tests/unit/core/primitives/test_interval.py

```diff
@@ -15,35 +15,211 @@
 
 import testtools
 
+from kmip.core import exceptions
+from kmip.core import primitives
 from kmip.core import utils
 
 
 class TestInterval(testtools.TestCase):
+    """
+    Test suite for the Interval primitive.
+    """
 
     def setUp(self):
         super(TestInterval, self).setUp()
-        self.stream = utils.BytearrayStream()
+
+        # Encoding and value taken from Section 9.1.2 of the KMIP 1.1
+        # specification.
+        self.value = 864000
+        self.encoding = (
+            b'\x42\x00\x00\x0A\x00\x00\x00\x04\x00\x0D\x2F\x00\x00\x00\x00'
```

Comment:
This might be nit-picking, but for the sake of clarity I would then suggest we say "based on". Thanks for the info though and for pointing out where the tags are explained, I was looking through the spec to see if I could find where the sections were explained :)